scania-digital-design-system/sdds
# fix(divider): migration of the divider component

**Describe pull-request**  
Migration on divider component

**Solving issue**  
Fixes: [AB#2402](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2402)

**How to test**  
1. Go to storybook page below.
2. Check in components -> divider
3. Check that the design reflects the one in Figma.

**Screenshots**  
-

**Additional context**  
-
